src/backend/marsha/core/api/video.py

```diff
@@ -76,14 +76,14 @@ def get_permissions(self):
             permission_classes = [
                 permissions.IsTokenResourceRouteObject
                 & (permissions.IsTokenInstructor | permissions.IsTokenAdmin)
-                | permissions.IsVideoPlaylistAdmin
```

Comment:
In your commit body

`s/to have specifi permissions/to have specific permissions`